packages/flutter/test/widgets/image_resolution_test.dart

```diff
@@ -259,46 +257,21 @@ void main() {
     expect(getRenderImage(tester, key).scale, 4.0);
   });
 
-  testWidgets('Image for device pixel ratio 1.0, with no main asset', (WidgetTester tester) async {
-    const String manifest = '''
-    {
-      "assets/image.png" : [
-        "assets/1.5x/image.png",
-        "assets/2.0x/image.png",
-        "assets/3.0x/image.png",
-        "assets/4.0x/image.png"
-      ]
-    }
-    ''';
-    final AssetBundle bundle = TestAssetBundle(manifest: manifest);
-
-    const double ratio = 1.0;
-    Key key = GlobalKey();
-    await pumpTreeToLayout(tester, buildImageAtRatio(image, key, ratio, false, images, bundle));
-    expect(getRenderImage(tester, key).size, const Size(200.0, 200.0));
-    expect(getRenderImage(tester, key).scale, 1.5);
-    key = GlobalKey();
-    await pumpTreeToLayout(tester, buildImageAtRatio(image, key, ratio, true, images, bundle));
-    expect(getRenderImage(tester, key).size, const Size(48.0, 48.0));
-    expect(getRenderImage(tester, key).scale, 1.5);
-  });
-
   testWidgets('Image for device pixel ratio 1.0, with a main asset and a 1.0x asset', (WidgetTester tester) async {
     // If both a main asset and a 1.0x asset are specified, then prefer
     // the 1.0x asset.
 
-    const String manifest = '''
+    final Map<dynamic, dynamic> manifest = json.decode('''
```

Comment:
Here and elsewhere, `Map<dynamic, dynamic>` is the same runtime type as `Map<Object?, Object?>`, however the latter will prevent accidentally making dynamic calls on either the keys/objects retrieved from the map. I would change the type signatures where possible

Reply:
This is part of our style guide too https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic